javascript/RagGrid.js

```diff
@@ -36,6 +36,12 @@ HTMLWidgets.widget({
         return {
 
             renderValue: function(x) {
+                if(document.location.search.indexOf("viewer_pane=1")!=-1 && navigator.platform==="Win32")
+                {
```

Comment:
Can you change the text to this ?

ag-grid uses CSS transform to position rows in the grid. It looks like your Rstudio Viewer Pane doesn't support this. 
--- line break ---
Open the <a href='"+document.location.href.replace("viewer_pane=1","")+"'>link</a> in other browser or use the Show in New Window button from the tools.